Autopilot/Inc/adc.h

```diff
@@ -10,7 +10,7 @@
   * inserted by the user or by software development tools
   * are owned by their respective copyright owners.
   *
-  * Copyright (c) 2017 STMicroelectronics International N.V. 
+  * Copyright (c) 2018 STMicroelectronics International N.V. 
```

Comment:
Really don't like dragging library updates along for the ride. Can this be a different PR?